src/static/js/routes/offices/index.js

```diff
@@ -0,0 +1,24 @@
+/* ==========================================================================
+   Scripts for `/offices/`.
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+var MobileOnlyExpandable = require( '../../modules/classes/MobileOnlyExpandable' );
+
+function init() {
+
+  // TODO: Remove this when per-page JS is introduced.
+  if ( document.querySelectorAll( '.office' ).length === 0 ) {
```

Comment:
So I think it should be the same speed because there will only be one element with the office class per page.